nupic/frameworks/opf/metrics.py

```diff
@@ -1459,7 +1424,63 @@ def addInstance(self, groundTruth, prediction, record = None):
 
     return self.aggregateError
 
+###################################################################################
+class MetricMulti(MetricsIface):
+  """Multi metric can combine multiple other (sub)metrics and 
+     weight them to provide combined score."""
 
+  def __init__(self, metricSpec):
+    """MetricMulti constructor using metricSpec is not allowed."""
+    raise ValueError("MetricMulti cannot be constructed from metricSpec string! "
+                     "Use MetricMulti(id,weights,metrics) constructor instead.")
+
+  def __init__(self, weights, metrics, id=None, window=None):
+    """MetricMulti 
+       @param id - unique name for this metric
```

Comment:
preparation for the next steps. in the `minimize=` field of `permutations.py` the MultiMetric cannot be described/constructed by the string, so I want to make it possible to identify the metric by `id`